### PR TITLE
Pal 544 k8s end to end

### PR DESCRIPTION
**Pull request by pd104923**

### CODE REVIEW

**Review by m78233 — DISMISSED** [review]

**Review by dev930018 — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -15,288 +15,331 @@
...
+                                if (sh(script: "kubectl get ns ${HELM_DEPLOY_NAMESPACE}", returnStatus: true) == 0) {
+                                    sh "kubectl delete ns ${HELM_DEPLOY_NAMESPACE}"
+                                }
+                                if (sh(script: "namespace-create ${HELM_DEPLOY_NAMESPACE}", returnStatus: true) == 0) {
+                                    if (sh(script: "bash deployment/aws-k8s/example-model/deployServicesToK8s.sh -n ${HELM_DEPLOY_NAMESPACE} -r ${ECR_REGISTRY} -h ${EGRESS_ELB} -d ${VOLUME_HANDLE_DATA_STORE} -c ${VOLUME_HANDLE_CLASSPATH_JARS}", returnStatus:
+                                    true) == 0) {
+                                        echo("successfully deployed")
+                                        sleep(time: 90, unit: 'SECONDS')
+                                        sh "bash deployment/aws-k8s/example-model/runFormattedK8sExample.sh ${HELM_DEPLOY_NAMESPACE}"
+                                        sh "bash deployment/aws-k8s/example-model/verify.sh ${HELM_DEPLOY_NAMESPACE}"
```
Comment (ac74475) [inline Jenkinsfile:326]: delete namespace if it succeeded and it is a feature branch. We will keep it up if it fails so developers can debug

**Review by ac74475 — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (pd104923) [reply Jenkinsfile:326]: Added `kubectl delete ns ${HELM_DEPLOY_NAMESPACE}` if successful

File: Jenkinsfile [hunk 2]
```diff
@@ -15,288 +15,332 @@
...
+                                        sh "kubectl delete ns ${HELM_DEPLOY_NAMESPACE}"
```
Comment (ac74475) [inline Jenkinsfile:327]:
```suggestion
                                        if (FEATURE_BRANCH == "true") {
                                            sh "kubectl delete ns ${HELM_DEPLOY_NAMESPACE}"
                                        }
```

**Review by ac74475 — CHANGES_REQUESTED** [review]

**Review by ac74475 — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (pd104923) [reply Jenkinsfile]: Updated

**Pull request merged**